---
 arch/arm/include/asm/io.h |    1 +
 1 file changed, 1 insertion(+)

Index: linux-3.12.24-rt37-r7s5/arch/arm/include/asm/io.h
===================================================================
--- linux-3.12.24-rt37-r7s5.orig/arch/arm/include/asm/io.h
+++ linux-3.12.24-rt37-r7s5/arch/arm/include/asm/io.h
@@ -327,6 +327,7 @@ extern void _memset_io(volatile void __i
  */
 #define ioremap(cookie,size)		__arm_ioremap((cookie), (size), MT_DEVICE)
 #define ioremap_nocache(cookie,size)	__arm_ioremap((cookie), (size), MT_DEVICE)
+#define ioremap_cache(cookie,size)	__arm_ioremap((cookie), (size), MT_DEVICE_CACHED)
 #define ioremap_cached(cookie,size)	__arm_ioremap((cookie), (size), MT_DEVICE_CACHED)
 #define ioremap_wc(cookie,size)		__arm_ioremap((cookie), (size), MT_DEVICE_WC)
 #define iounmap				__arm_iounmap
